content/docs/introduction/design-doc.md

```diff
@@ -45,6 +45,7 @@ still relevant.
 | [Extrapolation in range selectors (xrate)](https://docs.google.com/document/d/1y2Mp041_2v0blnKnZk7keCnJZICeK2YWUQuXH_m4DVc/edit#) | | Under review | |
 | Serverless, MQTT, and IoT use cases in the Prometheus ecosystem | | TODO | |
 | Static arithmetic for timestamps and durations | | TODO | |
+| [Prometheus Long-Term supported releases](https://docs.google.com/document/d/1wCXLnvh460UG10Mw019UbYMWDwSWWdvgwY61LHY0A0w/edit) | 2022-03-02 | Proposed | |
```

Comment:
For my OCD, could you make the capitalization consistent? E.g. "Prometheus long-term supported releases" would be most consistent with the other titles above.

If you want to emphasize the acronym, it could also be "Prometheus Long-Term Supported (LTS) releases".

Reply:
I don't want to emphasize the acronym since it means something else in the prometheus ecosystem already.